### PR TITLE
Tidy natural_disasters.py example

### LINKED ISSUE #2627: x-axis tick labels in Natural Disasters case study need clean up
See:

Screenshot from 2022-06-30 14-20-48

### PR DESCRIPTION
**Pull request by palewire**
Fixes #2627 

Before:
176780327-c5f2ee5d-69e8-46fe-8bb2-89e7e3843e7a

After:
visualization (1)

### CODE REVIEW

File: altair/examples/natural_disasters.py [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
-    alt.Color('Entity:N', legend=None)
+    color=alt.Color('Entity:N', legend=None),
+    tooltip=["Entity:N", "Year:O", "Deaths:Q"]
 ).properties(
     width=450,
-    height=320
-).transform_filter(
-    alt.datum.Entity != 'All natural disasters'
+    height=320,
+    title="Global Deaths from Natural Disasters"
```
Comment (mattijn) [inline altair/examples/natural_disasters.py:35]:
```suggestion
    title=alt.TitleParams(
        text="Global Deaths from Natural Disasters (1900-2017)",
        subtitle="The size of the bubble represents the total death count per year, by type of disaster",
        anchor='start'
    )
```

File: altair/examples/natural_disasters.py [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
-    strokeWidth=1
+    strokeWidth=1,
+    strokeOpacity=0.4
 ).encode(
-    alt.X('Year:O', axis=alt.Axis(labelAngle=0)),
-    alt.Y('Entity:N'),
-    alt.Size('Deaths:Q',
-        scale=alt.Scale(range=[0, 4000]),
-        legend=alt.Legend(title='Annual Global Deaths')
+    x=alt.X('Year:O', axis=alt.Axis(labelAngle=0), title=None),
```
Comment (mattijn) [inline altair/examples/natural_disasters.py:20]:
```suggestion
    x=alt.X('Year:T', title=None, scale=alt.Scale(domain=['1899','2018'])),
```

File: altair/examples/natural_disasters.py [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
+    y=alt.Y(
+        'Entity:N',
+        sort=alt.EncodingSortField(field="Deaths", op="sum", order='descending'),
+        title=None
+    ),
+    size=alt.Size('Deaths:Q',
+        scale=alt.Scale(range=[0, 2500]),
+        legend=alt.Legend(title='Deaths')
```
Comment (mattijn) [inline altair/examples/natural_disasters.py:28]:
```suggestion
        legend=alt.Legend(title='Deaths', clipHeight=30, format='s')
```

File: altair/examples/natural_disasters.py [hunk 4]
```diff
@@ -1,29 +1,43 @@
...
     ),
-    alt.Color('Entity:N', legend=None)
+    color=alt.Color('Entity:N', legend=None),
+    tooltip=["Entity:N", "Year:O", "Deaths:Q"]
```
Comment (mattijn) [inline altair/examples/natural_disasters.py:31]:
```suggestion
    tooltip=[
        "Entity:N", 
        alt.Tooltip("Year:T", format='%Y'), 
        alt.Tooltip("Deaths:Q", format='~s')
    ],
```

File: altair/examples/natural_disasters.py [hunk 5]
```diff
@@ -1,29 +1,43 @@
...
 ).properties(
     width=450,
-    height=320
-).transform_filter(
-    alt.datum.Entity != 'All natural disasters'
+    height=320,
+    title="Global Deaths from Natural Disasters"
+).configure_axis(
```
Comment (mattijn) [inline altair/examples/natural_disasters.py:36]:
```suggestion
).configure_axisY(
```

File: altair/examples/natural_disasters.py [hunk 6]
```diff
@@ -1,29 +1,43 @@
...
+    domain=False,
+    ticks=False,
```
Comment (mattijn) [inline altair/examples/natural_disasters.py:38]:
```suggestion
    ticks=False,
    offset=10
```

File: altair/examples/natural_disasters.py [hunk 7]
```diff
@@ -1,29 +1,43 @@
...
-    height=320
-).transform_filter(
-    alt.datum.Entity != 'All natural disasters'
+    height=320,
+    title="Global Deaths from Natural Disasters"
+).configure_axis(
+    domain=False,
+    ticks=False,    
+).configure_axisX(
+    labelOverlap=True,
```
Comment (mattijn) [inline altair/examples/natural_disasters.py:40]:
```suggestion
    grid=False,
```

**Review by mattijn — COMMENTED** [review]
Its great you are digging through all the examples! 
If we going to improve the enhancement of this chart, then maybe we can improve it even bit more (its a chart in the case-study section anyways).

See my in-line suggestions that you could accept to your liking.

If all changes are accepted it should resolve into the following:
image
I changed the following:
- Defined the x-axis as temporal, had to start the domain from 1899 to 2018 to get a tick at 1900 and a tooltip for 2017 values.
- Changed the format of legend and tooltip values.
- Provided a title in combination with a smaller sized subtitle.
- Included an offset on the y-axis labels so they won't overlap with the circles.

**Comment by palewire** [comment]
Sure. I trust your judgement. I accepted all your proposals.

**Pull request merged**